src/main/java/org/zalando/planb/provider/CassandraProperties.java

```diff
@@ -17,6 +19,8 @@
     private int port = ProtocolOptions.DEFAULT_PORT;
     private ConsistencyLevel writeConsistencyLevel = ONE; // TODO what is a sane default?
```

Comment:
Shouldn't we use `EACH_QUORUM` on write? See https://github.com/zalando/planb-revocation/issues/29